app/controllers/categories_controller.rb

```diff
@@ -5,9 +5,13 @@ class CategoriesController < ApplicationController
 
   def index
     @categories = Category.page(params[:page])
+
+    render json: @categories, only: %i[id name description]
```

Comment:
Changes are not related to this PR. You can rewrite the whole controller, or remove changes at all. We have a separate ticket for this.

Reply:
Done - the file is removed